alf/utils/averager.py

```diff
@@ -273,7 +274,7 @@ def update(self, tensor):
                 will be first averaged before being added to the average
         """
         self._update_ema_rate.fill_(
-            self._speed / self._total_steps.to(torch.float64))
+            self._speed / self._total_steps.to(torch.float32))
```

Comment:
why changing float64 to float32? If total_steps is huge, then there will be floating underflow.

Reply:
corrected